app.js

```diff
@@ -134,6 +134,7 @@ app.post('/join', passportConfig.isAuthenticated, teamController.postJoin);
 app.get('/create', passportConfig.isAuthenticated, teamController.getCreate);
 app.post('/create', passportConfig.isAuthenticated, teamController.postCreate);
 app.get('/finish', passportConfig.isAuthenticated, homeController.getFinish);
+app.get('Closed', passportConfig.isAuthenticated, homeController.getClosed);
```

Comment:
I would suggest to follow the same pattern as before i.e. change it to `/closed`.